scripts/restore

```diff
@@ -134,6 +132,10 @@ mkdir -p "$datadir"
 
 # Fix app ownerships & permissions
 chown -R $app: "$final_path" "$datadir"
+find $final_path/ -type f -print0 | xargs -0 chmod 0644
+find $final_path/ -type d -print0 | xargs -0 chmod 0755
+find $datadir/ -type f -print0 | xargs -0 chmod 0640
+find $datadir/ -type d -print0 | xargs -0 chmod 0750
```

Comment:
I think this commands with large quantity of files could break the script cause too much paths given to chmod :/

Reply:
It was already in this package and has been changed by an easier command

Reply:
It seems to work on sans-nuage... I will try on an other instance

Reply:
It works on several instances around 100GB of files